cli/dcoscli/service/main.py

```diff
@@ -3,30 +3,46 @@
 Usage:
     dcos service --info
     dcos service [--inactive --json]
+    dcos service log [--follow --lines=N --config-file=<path>]
+                     <service> [<file>]
     dcos service shutdown <service-id>
 
 Options:
-    -h, --help    Show this screen
+    -h, --help              Show this screen
 
-    --info        Show a short description of this subcommand
+    --info                  Show a short description of this subcommand
 
-    --json        Print json-formatted services
+    --config-file=<path>    Path to SSH config file.  Used to access marathon
+                            logs.
```

Comment:
I like to keep it consistent with the existing `dcos ... ssh` commands but let's consider renaming this to `--ssh-config-file`. Or if we allow them to change any part of the ssh command: `--ssh` or `--ssh-option`. See `mosh`: https://mosh.mit.edu/#usage